### ISSUE TITLE
research: is there a reason to set rx-buffers not to its maximum?

### ISSUE THREAD

**Issue by strizhechenko** [strizhechenko/netutils-linux#110]

**Comment by vincentbernat** [comment]
Yes, latency. Increasing buffers at every level leads to buffer bloat which heavily impacts latency. Small buffers are enough to catch short bursts while letting higher level protocols (eg TCP) handle congestion gracefully.

**Comment by strizhechenko** [comment]
Hi @vincentbernat.
I added few details about it, you absolutely right in cases of just forwarding/receiving traffic into an application.
